pkgs/applications/graphics/akira/default.nix

```diff
@@ -0,0 +1,40 @@
+{ stdenv, lib, fetchFromGitHub
+, desktop-file-utils, meson, ninja, pkgconfig, vala_0_40, wrapGAppsHook
+, appstream-glib, cairo, glib, goocanvas2, gtk3, gtksourceview3
+, json-glib, libarchive, libgee, libxml2, pantheon, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "akira";
+  version = "2019-10-12";
+
+  src = fetchFromGitHub {
+    owner = "akiraux";
+    repo = "Akira";
+    rev = "cab952dee4591b6bde34d670c1f853f5a3ff6b19";
+    sha256 = "1fp3a79hkh6xwwqqdrx4zqq2zhsm236c6fhhl5f2nmi108yxz04q";
+  };
+
+  nativeBuildInputs = [ desktop-file-utils meson ninja pkgconfig vala_0_40 wrapGAppsHook ];
+  buildInputs = [
+    appstream-glib cairo glib goocanvas2 gtk3 gtksourceview3
+    json-glib libarchive libgee libxml2 meson pantheon.granite python3
+  ];
+
+  mesonFlags = [ "-Dprofile=default" ];
+
+  patches = [ ./meson-post-install.patch ];
+
+  postInstall = ''
+    ${glib.dev}/bin/glib-compile-schemas --strict $out/share/glib-2.0/schemas
+    ${gtk3.out}/bin/gtk-update-icon-cache -qtf $out/share/icons/hicolor
+    ${desktop-file-utils.out}/bin/update-desktop-database $out/share/applications
+  '';
+
+  meta = with lib; {
+    description = "Native Linux Design application built in Vala and GTK";
+    homepage = "https://github.com/akiraux/Akira";
+    license = licenses.gpl3;
+    maintainers = with maintainers; [ filalex77 ];
```

Comment:
This app is a elementary application :sparkles: Can you add `pantheon.maintainers`. (it's currently just me).